servers/src/main/java/tachyon/worker/TachyonWorker.java

```diff
@@ -47,10 +56,18 @@ public static void main(String[] args) {
 
     try {
       worker.process();
+      // Start the lineage worker
+      if (LineageUtils.isLineageEnabled(WorkerContext.getConf())) {
+        lineageWorker.start();
```

Comment:
I can move lineageWorker.start() before worker.process(). In terms of moving web server and thrift server, looks like web server depends on blockDataManager, which in turn also depends on workerSource, blockMasterClient and fileSystemMasterClient. Do we really want to pull all these into workerBase?